explorer/client/README.md

```diff
@@ -14,33 +14,9 @@ $ pnpm install
 
 # How to Switch Environment
 
-## Connecting to the DevNet Remote Gateway Server
+By default, the Sui Explorer will attempt to connect to a locally running RPC server. Refer to [Local RPC Server & JSON-RPC API Quick Start](../../doc/src/build/json-rpc.md) on setting up a Local RPC Server.
```

Comment:
```suggestion
By default, the Sui Explorer attempts to connect to a local RPC server. For more information about using a  local RPC server, see [Local RPC Server & JSON-RPC API Quick Start](../../doc/src/build/json-rpc.md).
```

Reply:
doesn't it default to Devnet?

Reply:
This is just for when running the explorer locally. The deployed one defaults to whatever env it's deployed to (e.g explorer.devnet.sui.io points to devnet, explorer.mainnet.sui.io would point to mainnet)